docs/examples/spam/meson.build

```diff
@@ -3,39 +3,30 @@
 # SPDX-License-Identifier: MIT
 
 project(
-  'spam',
-  'c',
+    'spam',
+    'c',
```

Comment:
Meson's recommended style (what it uses in all its docs is 2 spaces), not 4. So the formatter seems to be in the wrong here.

Reply:
Being is used in the docs and being recommended are two very different things. I could not find anything relating to 2 vs 4 spaces in the actual [Meson style recommendations page](https://mesonbuild.com/Style-guide.html), so I would not say this style is "wrong".

At the end of the day, it doesn't really matter anyway.

Reply:
The indentation is configurable.

Reply:
I meant it's used by Meson everywhere, and also by us and other initial adopters like SciPy. So "felt" like a recommendation.

> At the end of the day, it doesn't really matter anyway.

Agreed. I'll resolve this comment - doesn't really matter.